consensus/babe/babe.go

```diff
@@ -46,17 +47,27 @@ type Session struct {
 	epochThreshold *big.Int // validator threshold for this epoch
 	txQueue        *tx.PriorityQueue
 	isProducer     map[uint64]bool // whether we are a block producer at a slot
+
+	// Block announce channel every time is creates a block
+	blockAnnounceChan chan<- p2p.BlockAnnounceMessage
```

Comment:
Please remove `chan` from the `blockAnnounceChan`. For idiomatic go you should avoid putting the type into the name